libs/domain/user/account-link/account-link.component.ts

```diff
@@ -0,0 +1,30 @@
+import { AuthService } from '@spryker-oryx/auth';
+import { resolve } from '@spryker-oryx/di';
+import { ButtonType } from '@spryker-oryx/ui/button';
+import { IconTypes } from '@spryker-oryx/ui/icon';
+import { hydrate, I18nMixin, signal } from '@spryker-oryx/utilities';
+import { html, LitElement, TemplateResult } from 'lit';
+import { styles } from './account-link.styles';
+
+@hydrate({ event: 'window:load' })
+export class AccountLinkComponent extends I18nMixin(LitElement) {
```

Comment:
Can we use just content-link instead of this specific component?

Reply:
I gave it a try. Unfortunately, the content link button style which is closest in appearance gives me no way to get rid of the box shadow that appears on hover.

Without the button style, the icon is the wrong size.

Should we make the box shadow color css configurable?